GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs

```diff
@@ -13,6 +13,8 @@ namespace GVFS.FunctionalTests.Tests.GitCommands
     [TestFixture]
     public abstract class GitRepoTests
     {
+        public const string ValidateWorkingTree = "WorkTreeValidation";

```

Comment:
Is this redirection buying us anything? Can we just use `nameof(WorkTreeValidation)` in all of the callers?

And actually, why not just reference `TestFixtureSource(typeof(GVFSTestConfig), nameof( GVFSTestConfig.GitCommandTestWorkTreeValidation)` directly?